src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -95,7 +95,7 @@ function onToggleLayerVisibility() {
 function onOpacityChange(e) {
     store.dispatch('setLayerOpacity', {
         index: index.value,
-        opacity: e.target.value,
+        opacity: 1.0 - e.target.value,
         ...dispatcher,
     })
 }
```

Comment:
The name of the function needs to be changed as it is not opacity but transparency, otherwise the computation don't make sense for reader.